Tests/js/views/assets/ez-studiopresentationview-tests.js

```diff
@@ -28,6 +30,31 @@ YUI.add('ez-studiopresentationview-tests', function (Y) {
             this.view.render();
             Y.Assert.isTrue(templateCalled, "The template should have been used to render this.view");
         },
+
+        "Test height fit": function () {
+            var container = this.view.get('container');
+            this.view.render();
+            this.view.set('active', true);
+
+            Y.Assert.areSame(
+                container.get('winHeight') - container.getY() + "px",
+                container.getStyle('height'),
+                "The height of the container should have been adapted to the windows size"
+            );
+        },
+
+        "Test iframe link": function () {
+            var iframeSource = this.view.get('container').one('.ez-studiopresentation-content').get('src');
+            this.view.render();
+
+            Y.Assert.isTrue(
```

Comment:
ok get it, you fetch the iframeSource before the view is rendered and since `tearDown` does not reset the container, you get the iframeSource of the previous test...